baseplate/sidecars/file_age_watcher.py

```diff
@@ -0,0 +1,49 @@
+"""Calculate the age of files sync'd to disk by the baseplate live-data watcher.
+
+The watcher periodically updates the mtime even if data hasn't changed, so we
```

Comment:
this says "periodically" but it only runs once. do you intend this to be a one-shot sidecar that gets spun up periodically by the hosting environment or for the sidecar to run indefinitely and manage its own cycle of checking file health?